_includes/comments.html

```diff
@@ -0,0 +1,60 @@
+<div class="row">
+    <div class="center">
+        <div class="col-md-3">
+            <div class="center">
+                <h2 class="no-anchor">Comments </h2>
+            </div>
+            <div class="center share-buttons">
+                <ul>
+                    {% capture url %}{{site.url}}{{page.url}}{% endcapture %}
+                    <li>
+                        <a  href="https://www.facebook.com/sharer/sharer.php?u={{ url }}"
+                        onclick="window.open(this.href, 'pop-up', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"
```

Comment:
Will this be blocked by pop-up blockers? How does it compare to `target="_blank"`?

Reply:
No it doesn't get blocked by pop-up blockers. But I'll also prefer opening in a different tab so the user gets to properly see how it looks before sharing. I wasn't sure how best to implement it.

Reply:
My guess is that a an `<a target="_blank">` is a better fit here than `window.open`. Make sure to add `rel="noopener noreferrer"` too.

Reply:
For context, see https://developers.google.com/web/tools/lighthouse/audits/noopener